dandi/tests/fixtures.py

```diff
@@ -63,6 +64,35 @@ def simple2_nwb(simple1_nwb_metadata, tmpdir_factory):
     )
 
 
+@pytest.fixture()
+def organized_nwb_dir(simple2_nwb, tmp_path, clirunner):
+    shutil.copy(str(simple2_nwb), str(tmp_path))
+    (tmp_path / dandiset_metadata_file).write_text("{}\n")
+    r = clirunner.invoke(organize, ["-f", "move", "--dandiset-path", str(tmp_path)])
```

Comment:
Coughs use `organize -f copy` instead of explicit shutil.copy above

Reply:
The goal of the fixture is to simply to create a temporary directory containing organized NWB files, a `dandiset.yaml`, and nothing else.  Not mixing in the `simple*.nwb` files at the root of the dandiset makes it cleaner.

Reply:
`-f copy` would just do that shutil.copy from their original location into properly named organized files. You just need to point organize command to original paths. Or am I missing the point?

Reply:
Where would it put the organized files?  Next to the originals (not what I want) or in the current directory?

Reply:
It should place into the specified dandiset path, otherwise should assumr that curdir is the dandiset path. Dude might need time up if that is not said in --help